extensions/kill-process/src/index.tsx

```diff
@@ -117,20 +172,46 @@ export default function ProcessList() {
           return (
             <List.Item
               key={index}
-              title={process.name}
+              title={(process.type == "aggregatedApp" ? process.appName : process.processName) ?? ""}
```

Comment:
I don't understand how would this fallback on `?? ""`, isn't `process.type == "aggregatedApp" ? process.appName : process.processName` enough?